### DISCUSSION TITLE
LG Aircon behavior in Home app

### DISCUSSION THREAD

**Discussion by remoscq** [nVuln/homebridge-lg-thinq#236] [Q&A]
Hello!

i apologize if this is a stupid question but can anyone share with me how does their LG air conditioners behave and appear inside of the home app? All I really want to know is what device appears and how the device is controlled, AFAIK Homekit has native AC support, so does it show it as an AC?

I looked into some native solutions and I found the [Sensibo Air](https://sensibo.com/products/sensibo-air), however import taxes makes it cost prohibitive for me at the moment, but I love the way it shows up as a thermostat.

If the LG AC doesn't appear as a thermostat is there anything I can do myself to change that?
again im so sorry if this sounds ignorant or dumb, I really couldn't find much about it on google, simply sharing a screenshot of how the AC shows up in the home app would be of great help!

Thank you so much for sparing some time to read this <3
Edit - I apologize for any mistakes in my grammar or wording, English is not my first language.

**Comment by nVuln** [comment]
did you tried install homebridge and this plugin before?

**Comment by remoscq** [comment]
Yep, i do have home bridge running, not the plug-in though since my AC is still in shipping, all I want to know is just how the AC appears in HK, is it a thermostat, an AC, or a switch?

**Reply by nVuln** [reply]
yep, it's show as AC, sorry I don't have AC right now to capture some screenshot for you

**Comment by remoscq** [comment]
Thank you so much!Is it okay if I change the code for myself to make it appear as a thermostat? I believe that would be better for my use case and better device to control an ACEm 8 de mai. de 2023, à(s) 12:56 PM, nVuln ***@***.***> escreveu:﻿
yep, it's show as AC, sorry I don't have AC right now to capture some screenshot for you

**Reply by nVuln** [reply]
yup, just fork this repo and custom whatever you want, message me on discord if you need help

**Comment by remoscq** [comment]
Will do! Thanks a lot :)Enviado do meu iPhoneEm 10 de mai. de 2023, à(s) 10:25 PM, nVuln ***@***.***> escreveu:﻿
yup, just fork this repo and custom whatever you want, message me on discord if you need help
